mitosis.config.cjs

```diff
@@ -20,7 +20,7 @@ const VUE_OPTIONS = {
   api: 'composition',
 };
 
-const components = ['Login', 'CreateOIDCConnection', 'CreateSAMLConnection', 'ConnectionList'];
+const components = ['Login', 'CreateOIDCConnection', 'CreateSAMLConnection'];
```

Comment:
Is this intentional?

Reply:
Yup the plugin is for angular and it fails since I cleaned up some stuff.